src/components/wallet/methods/depositAsset.ts

```diff
@@ -44,14 +44,17 @@ export default async function depositAsset(
       }
       homeDomain = inputs[0].value
     }
-
-    homeDomain = homeDomain.startsWith('https://')
-      ? homeDomain
-      : 'https://' + homeDomain
+    homeDomain =
+      homeDomain.startsWith('http://') || homeDomain.startsWith('https://')
```

Comment:
This is explicit so we could keep it, but you could just do:
```suggestion
      homeDomain.startsWith('http')
```
Since this will return `true` for domains with `https` and `http`.

Reply:
Good catch!